### PR TITLE
Extract chip component for better reusability

### PR DESCRIPTION
**Pull request by danrot**
| Q | A
| --- | ---
| Bug fix? | no
| New feature? | yes
| BC breaks? | no
| Deprecations? | no
| Fixed tickets | ---
| Related issues/PRs | ---
| License | MIT
| Documentation PR | ---

#### What's in this PR?

This PR is extracted from #5035 and contains a new `Chip` component.

#### Why?

Because the `Chip` component will also be used when filtering lists. Until now this component was only available to the `MultiAutoComplete`.

### CODE REVIEW

**Comment by danrot** [comment]
Includes ee40522c5b7224697c320692188899db14eb0b3e and dfa3d095c6495e468b25379debf7c0496b52125d from #5035.

**Comment by nnatter** [comment]
i think this pr contains unrelated changes right now

**Comment by nnatter** [comment]
should we target the `release/2.0` branch here? otherwise we might end up with unnecessary merge conflicts in the future. this is not a new feature imo

**Comment by danrot** [comment]
The unrelated changes will be gone as soon as I push the `master` branch :slightly_smiling_face: 

> should we target the `release/2.0` branch here? otherwise we might end up with unnecessary merge conflicts in the future. this is not a new feature imo

But it not a bugfix either... I think I would leave it as it is, because it might have potential bc breaks, and I don't think that this component will change a lot in 2.0...

File: src/Sulu/Bundle/AdminBundle/Resources/js/components/Chip/tests/Chip.test.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
+// @flow
+import React from 'react';
+import {render, shallow} from 'enzyme';
+import Chip from '../Chip';
+
+test('Should render item with children', () => {
+    expect(render(<Chip value={{}}>Name</Chip>)).toMatchSnapshot();
+});
+
+test('Should render item with delete icon', () => {
```
Comment (nnatter) [inline src/Sulu/Bundle/AdminBundle/Resources/js/components/Chip/tests/Chip.test.js]: should we add a test case to check if the item is rendered `clickable` if an `onClick` callback is giben?

File: src/Sulu/Bundle/AdminBundle/Resources/js/components/Chip/tests/Chip.test.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    expect(render(<Chip onDelete={jest.fn()} value={{}}>Name</Chip>)).toMatchSnapshot();
+});
+
+test('Should render item without icon in disabled state', () => {
```
Comment (nnatter) [inline src/Sulu/Bundle/AdminBundle/Resources/js/components/Chip/tests/Chip.test.js]: `without delete icon`

**Pull request merged**